jetson/nav/search/searchStateMachine.cpp

```diff
@@ -275,13 +275,13 @@ NavState SearchStateMachine::executeDriveToTarget( Rover* phoebe, const rapidjso
         {
             roverStateMachine->mGateStateMachine->mGateSearchPoints.clear();
             const double absAngle = mod(phoebe->roverStatus().odometry().bearing_deg +
```

Comment:
spacing around parenthesis not consistent with rest of code

please do this throughout